README.md

```diff
@@ -1,62 +1,64 @@
-source{d} MLoSC models
-======================
+source{d} MLonCode models
+=========================
 
-## docfreq
-Document frequencies of source code identifiers, that is, how many projects contain an identifier.
+## bow
+Weighted bag-of-words, that is, every bag is a feature extracted from source code and associated with a weight obtained by applying TFIDF.
 
 Example:
 
 ```
-from ast2vec import DocumentFrequencies
-df = DocumentFrequencies().load()
-print("Number of tokens:", len(df))
+from sourced.ml.models import BOW
+bow = BOW().load(bow)
+print("Number of documents:", len(bow))
+print("Number of tokens:", len(bow.tokens))
 ```
 
 1 model:
 
-* <default> [f64bacd4-67fb-4c64-8382-399a8e7db52a](/docfreq/f64bacd4-67fb-4c64-8382-399a8e7db52a.md) 
+* <default> [1e3da42a-28b6-4b33-94a2-a5671f4102f4](/bow/1e3da42a-28b6-4b33-94a2-a5671f4102f4.md)
```

Comment:
what are these for?
there's no documentation explaining what these numbers mean

Reply:
Each model has a UUID. Also each type has a default UUID. Indeed, we should document this.
Roughly the same way docker images work, "latest" is "default" here, and sha256s are UUIDs.

Reply:
done: https://github.com/src-d/modelforge/pull/44